pkg/cmd/community.go

```diff
@@ -0,0 +1,52 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/stripe/stripe-cli/pkg/open"
+)
+
+const communityURL = "https://stripe.com/go/developer-chat"
+
+var openBrowser = open.Browser
+var canOpenBrowser = open.CanOpenBrowser
+
+type communityCmd struct {
+	cmd *cobra.Command
+}
+
+func newCommunityCmd() *communityCmd {
+	cc := &communityCmd{}
+
+	cc.cmd = &cobra.Command{
+		Use:     "community",
+		Short:   "Chat with Stripe engineers and other developers",
+		Long:    "Chat with Stripe engineers and other developers in the official Stripe Developer Community Discord server",
+		Example: "stripe community",
+		RunE:    cc.runCommunityCmd,
+	}
+
+	return cc
+}
+
+func (cc *communityCmd) runCommunityCmd(cmd *cobra.Command, args []string) error {
+	if !canOpenBrowser() {
+		fmt.Printf("Chat with other developers and Stripe engineers in the official Stripe Discord server: %s\n", communityURL)
+		return nil
+	}
+
+	fmt.Printf("Chat with other developers and Stripe engineers in the official Stripe Discord server.\n\nPress Enter to open the browser or visit %s", communityURL)
+
+	input := os.Stdin
+	fmt.Fscanln(input)
+
+	err := openBrowser(communityURL)
+	if err != nil {
+		fmt.Printf("Failed to open browser, please go to %s manually.", communityURL)
```

Comment:
We should probably return an error here instead of printing it.

Reply:
I was copying the behaviour here: https://github.com/stripe/stripe-cli/blob/6e96f25437fc9e3a26924a49e9e970c331c6ecea/pkg/login/client_login.go#L66-L71

Reply:
Happy to return an error though instead if you feel that's more appropriate.

Reply:
Ah alright. That's fine for now!